schema.json

```diff
@@ -158,50 +173,164 @@
                         "32bit": {
                             "additionalProperties": false,
                             "properties": {
-                                "extract_dir": {
-                                    "type": "string"
-                                },
                                 "url": {
-                                    "format": "uri",
-                                    "type": "string"
+                                    "$ref": "#/definitions/autoupdateUriOrArrayOfAutoupdateUris"
                                 },
                                 "hash": {
-                                    "$ref": "#/definitions/hashExtraction"
+                                    "$ref": "#/definitions/hashExtractionOrArrayOfHashExtractions"
+                                },
+                                "extract_dir": {
```

Comment:
All those properties could be moved to a single definition, I guess. 🤔

Reply:
Move properties under architecture to single definition, but those under autoupdate, I've no idea...